experiments/src/browsergym/experiments/loop.py

```diff
@@ -290,13 +291,16 @@ class StepInfo:
     agent_info: dict = field(default_factory=dict)
     stats: dict = None
     profiling: StepTimestamps = field(default_factory=StepTimestamps)
+    task_info: dict = None
```

Comment:
Why not just adding env_info here? If we put more stuff in there in the future, it will be readily available without the need to patch StepInfo.

Reply:
talked offline